Find a functionally equivalent code:android.media.MediaCodecInfo.getName:COMMENT
Method Modifier: public      final       
Comment:/**
 * Retrieve the codec name.
 */

@@ -1,3 +1,3 @@
 {
-    return MediaCodecList.getCodecName(mIndex);
+    return mName;
 }

